build.sbt

```diff
@@ -3,6 +3,7 @@ name := "oni-ml"
 version := "1.1"
 
 scalaVersion := "2.10.5"
+scalacOptions += "-target:jvm-1.7"
```

Comment:
specifying a version of the jvm.... 7

It was meant to fix an issue but it didn't so I should clean it out